includes/reader-activation/sync/class-metadata.php

```diff
@@ -7,6 +7,8 @@
 
 namespace Newspack\Reader_Activation\Sync;
 
+use Error;
```

Comment:
What's this for?

Reply:
That's a mistake—my VSCode likes to "helpfully" try to insert dependencies if it thinks I'm missing any (such as if I'm typing `error_log`)